src/bindings/python/fluxacct/accounting/user_subcommands.py

```diff
@@ -158,6 +158,24 @@ def delete_user(conn, username, bank):
     # commit changes
     conn.commit()
 
+    # check if bank being deleted is the user's default bank
+    cur = conn.cursor()
+    select_stmt = "SELECT default_bank FROM association_table WHERE username=?"
+    cur.execute(select_stmt, (username,))
+    rows = cur.fetchall()
+    default_bank = rows[0][0]
+
+    if default_bank == bank:
+        # get first bank from other potential existing rows from user
+        select_stmt = """SELECT bank FROM association_table WHERE active=1 AND username=?
+                         ORDER BY creation_time"""
+        cur.execute(select_stmt, (username,))
+        rows = cur.fetchall()
+        # update user rows to have a new default bank (the next earliest user/bank row created)
+        if len(rows) > 0:
+            new_default_bank = rows[0][0]
+            edit_user(conn, username, default_bank=new_default_bank)
```

Comment:
what if the user is not a member of any other bank?  should default_bank be set to `None` or something like that?

and would test in follow on test be needed?

Reply:
I'm sorry, I think I am just confused by your question and am probably not answering what you are actually asking. If a user only belongs to one bank (thus, their default bank is the same as the bank they belong to), and the user gets disabled, then their default bank does not need to be updated at all. It will keep the same default bank it had before the user got disabled because the user does not belong to any other banks. Does that make sense? :P

Reply:
ohh ok, that makes sense :-)  I'll let ya decide if a comment is warranted, maybe I'm just a dummy :-)

Reply:
Haha no you're not - I probably just didn't do a good job explaining it. I'll add a detailed comment just above explaining this case so that if/when other people look at it they won't raise similar questions.